From: Matthijs Kooijman <matthijs@stdin.nl>
Subject: Use a wrapper script for running openttd

The wrapper script captures stderr from openttd and displays this in
case of an error. This patch makes the the .desktop file call the
wrapper instead of the openttd binary directly.

Index: media/openttd.desktop.in
===================================================================
--- a/media/openttd.desktop.in	(revision 20124)
+++ b/media/openttd.desktop.in	(working copy)
@@ -5,7 +5,7 @@
 Version=1.1
 Name=!!MENU_NAME!!
 Icon=openttd
-Exec=!!TTD!!
+Exec=/usr/share/games/openttd/openttd-wrapper
 Terminal=false
 Categories=!!MENU_GROUP!!
 Comment=A clone of Transport Tycoon Deluxe
